dbms/src/Flash/tests/gtest_window_executor.cpp

```diff
@@ -71,13 +71,15 @@ class WindowExecutorTestRunner : public DB::tests::ExecutorTest
         WRAP_FOR_DIS_ENABLE_PLANNER_END
     }
 
-    void executeWithTableScanAndConcurrency(const std::shared_ptr<tipb::DAGRequest> & request, const ColumnsWithTypeAndName & source_columns, const ColumnsWithTypeAndName & expect_columns)
+    void executeWithTableScanAndConcurrency(const std::shared_ptr<tipb::DAGRequest> & request, const String & db, const String & table_name, const ColumnsWithTypeAndName & source_columns, const ColumnsWithTypeAndName & expect_columns)
     {
+        context.addMockTableColumnData(db, table_name, source_columns);
+
         WRAP_FOR_DIS_ENABLE_PLANNER_BEGIN
-        ASSERT_COLUMNS_EQ_R(expect_columns, executeStreamsWithSingleSource(request, source_columns, SourceType::TableScan));
+        // ASSERT_COLUMNS_EQ_R(expect_columns, executeStreams(request, SourceType::TableScan));
```

Comment:
why "//"?

Reply:
> why "//"?

Ops, just a temporary comment, I will remove it.